api/test/image_integration_test.py

```diff
@@ -53,6 +54,11 @@ def test_search_quotes():
     search_quotes("images", "dog")
 
 
+def test_search_quotes_exact():
+    # ``bird perched`` returns different results when quoted vs unquoted
+    search_quotes_exact("images", "bird perched")
```

Comment:
```suggestion
    search_quotes_exact("images", "fireworks celebration")
```

This should make the tests pass!

Reply:
Just to document why this wasn't committed, taking a slightly different approach described here: https://github.com/WordPress/openverse-api/pull/1012#issuecomment-1330019285